extension/lib/position.js

```diff
@@ -1,52 +1,50 @@
 let $ = require('jquery');
 
-let isImage = function(element) {
-  return element.prop('tagName') === 'IMG';
-};
-
-let setPositionValue = function(position, key, value) {
+let setObjectValue = function(object, key, value) {
   if (value) {
-    position[key] = value;
+    object[key] = value;
   }
 
-  return position;
+  return object;
 };
 
-let constructPosition = function(callback) {
+let construct = function(callback) {
   let url      = 'https://localhost/api/v1/selectors';
   let promise  = $.getJSON(url, {
     host: location.host
   });
 
   promise.done(function(json, status, xhr) {
-    let position = Object.keys(json.data).reduce(function(position, key) {
-      let elements = $(json.data[key]);
+    let object = Object.keys(json.data).reduce(function(object, key) {
```

Comment:
The key is what we are attempting to structure, the selector is the css selector to the dom node, and the attribute is the attribute on the dom node that returns the information.

I changed it to be more flexible. Here is an example...

IMG -> img.src
META -> meta.content
H1 -> h1.innerText
INPUT -> input.value

They are all different attributes, but the existing selector structure would have required the code to include a lot of if statements to handle "types" of nodes in different ways. In the way this is written, we can now offload that condition logic into the structure of the selector and make the code use that logic directly.